lib/src/main/java/graphql/nadel/NadelDefinitionRegistry.kt

```diff
@@ -78,6 +78,10 @@ class NadelDefinitionRegistry {
         return sdlDefinitions.filterIsInstance<T>()
     }
 
+    fun getDefinitions(name: String): List<AnySDLDefinition> {
+        return definitionsByName[name] ?: return emptyList()
```

Comment:
Allows for faster lookups.

Some code was doing `.definitions.filter { it.name == name }`